static/js/components/Toast.js

```diff
@@ -13,24 +12,24 @@ export default class Toast extends React.Component {
   };
 
   componentDidMount() {
-    const { onTimeout, open, timeout } = this.props;
+    const { onTimeout, timeout } = this.props;
 
-    if (open && onTimeout) {
+    if (onTimeout) {
       setTimeout(onTimeout, timeout);
     }
   }
 
-  componentDidUpdate(prevProps: Object) {
+  componentDidUpdate() {
     const { onTimeout, timeout } = this.props;
-    if (!prevProps.open && this.props.open && onTimeout) {
+    if (onTimeout) {
       setTimeout(onTimeout, timeout);
     }
```

Comment:
Well, my suspicion is that we could run into an edge case where componentDidUpdate() is called multiple times - I don't know how much of a concern that would be. But, in any case, I think because we are now render the component on-demand we can just lean on componentDidMount and delete the componentDidUpdate method.